src/user/dto/create-user.dto.ts

```diff
@@ -0,0 +1,4 @@
+export class CreateUserDto {
```

Comment:
Can this be `DTO` instead of `Dto` as it's an acronym (I know Nest automatically formats it as `Dto`) please!

```suggestion
export class CreateUserDTO {
```

Reply:
Done